test/lock_test.rb

```diff
@@ -178,6 +178,14 @@ def test_queue_inspection
     assert !SlowJob.enqueued?, 'no loner lock key should hae been created'
   end
 
+  def test_loner_job_locked_depending_on_args
+    assert Resque.enqueue(LonelyWithArgsJob)
+    assert Resque.enqueue(LonelyWithArgsJob), 'should enqueue if arg not given'
```

Comment:
@amiryal I know its a bit to ask, because it's been a while, but I'd appreciate some input from you so we can see this addressed.

From the "loner" docs it says:

```Two jobs of the same class with the same parameters/arguments/workload must produce the same redis_key```

Does this not mean the 2nd enqueue should fail? and we should have 2 jobs enqueued total?